README.md

```diff
@@ -10,7 +10,10 @@ Gitops native package management
 1. deploy an example catalog `kubectl apply -f examples/profile-catalog-source.yaml`
 
 1. To query the catalog API run `kubectl -n profiles-system port-forward <profiles-controller-pod-name> 8000:8000` to enable access to the API and use
-[pctl](https://github.com/weaveworks/pctl) to query, for example: `pctl search --catalog-url http://localhost:8000 <profile-name>`
+[pctl](https://github.com/weaveworks/pctl) to query, for example: `pctl --catalog-url http://localhost:8000 show <search-string>`
+
+1. To see more details on a specific Profile in the catalog, use
+[pctl](https://github.com/weaveworks/pctl): `pctl --catalog-url http://localhost:8000 show <profile-name>`
```

Comment:
same nit as in https://github.com/weaveworks/pctl/pull/8 lets use the environment variable for CATALOG_URL, makes it eaisier to read :smile: